src/ClusterBootstrap/services/logging/elasticsearch.yaml

```diff
@@ -1,71 +1,117 @@
-# Elasticsearch deployment itself
-kind: DaemonSet
 apiVersion: apps/v1
+kind: DaemonSet
 metadata:
-  name: elasticsearch-logging
+  name: elasticsearch
   namespace: kube-system
   labels:
-    k8s-app: elasticsearch-logging
-    version: v5.6.2
+    k8s-app: elasticsearch
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
   selector:
     matchLabels:
-      k8s-app: elasticsearch-logging
-      version: v5.6.2
+      k8s-app: elasticsearch
   template:
     metadata:
       labels:
-        k8s-app: elasticsearch-logging
-        version: v5.6.2
-        kubernetes.io/cluster-service: "true"
+        k8s-app: elasticsearch
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/path: '/metrics'
+        prometheus.io/port: '{{cnf["elasticsearch"]["port"]["exporter"]}}'
     spec:
       hostNetwork: true
       nodeSelector:
         elasticsearch: active
+      initContainers:
+      - name: elasticsearch-sysctl
+        image: alpine:3.9
+        command:
+        - "/sbin/sysctl"
+        - "-w"
+        - "vm.max_map_count=262144"
+        securityContext:
+          privileged: true
+      - name: elasticsearch-chown
+        image: alpine:3.9
+        command:
+        - "chown"
+        - "-R"
+        - "1000:1000"
+        - "/usr/share/elasticsearch/data"
+        volumeMounts:
+        - name: elasticsearch
+          mountPath: /usr/share/elasticsearch/data
       containers:
-      - image: mlcloudreg.westus.cloudapp.azure.com:5000/dlworkspace/elasticsearch
-        imagePullPolicy: Always
-        name: elasticsearch-logging
+      - name: elasticsearch
+        image: docker.elastic.co/elasticsearch/elasticsearch:6.8.5
```

Comment:
please change the docker image to configurable parameter. Default value can be found in params.py